packages/skill-api/src/client/index.ts

```diff
@@ -1 +1,3 @@
 export * from './send-feedback'
+export * from './get-decoded-token'
+export * from './get-feedback-emoji'
```

Comment:
moved these to `client` because where `lib` is more like `server` and should likely be named as such

Reply:
I appreciate this context comment 👍